functions/src/notification/index.ts

```diff
@@ -0,0 +1,68 @@
+import * as functions from 'firebase-functions';
+import * as admin from 'firebase-admin';
+import * as Enumerable from 'linq';
+
+export const accountCreate = functions.auth.user().onCreate(user => {
+    const email = user.email;
+    const db = admin.database();
+    const ref = db.ref('/Email_list');
+    return ref.push(email);          
+});
+
+const sgMail = require('@sendgrid/mail');
+const API_KEY = functions.config().sendgrid.key;
+sgMail.setApiKey(API_KEY);
+
+export const notifyNewRelease = functions.database.ref('/Bulletin_Version/{bulletinid}').onCreate((snap,context) => {
```

Comment:
That won't work, because we rebuild the tree each time, correct?

For now we can just create a standalone function, taking bulletin id, and later we will call it from the function that processes the new bulletin data.